aio_pika/tools.py

```diff
@@ -45,8 +45,15 @@ def create_task(
 ) -> Awaitable[T]:
     loop = loop or asyncio.get_event_loop()
 
+    def _task_done(future: asyncio.Future) -> None:
```

Comment:
It's better to move this helper outside, - there is no need to define it for every task.